src/lang-pt_BR.js

```diff
@@ -0,0 +1,517 @@
+(function (janela) {

```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected linebreaks to be 'LF' but found 'CRLF'.](https://app.codacy.com/gh/congrea/virtualclass/pullRequest?prid=5482209)